kernel/pre_env.mli

```diff
@@ -47,7 +47,7 @@ type named_context_val = private {
 
 type env = {
     env_globals       : globals;
-    env_named_context : named_context_val;
+    env_named_context : named_context_val;  (** in the interactive theorem proving mode, we see these above ============================ *)
```

Comment:
I doubt this is the proper place to document this, `pre_env` is an internal structure of `Environ.env`. Worse, it is tied to the way we display things in coqtop, which is totally unconnected to the kernel. You should probably remove that comment and describe the structure in a separate document.